sdk/core/System.BinaryData/api/System.BinaryData.netstandard2.0.cs

```diff
@@ -0,0 +1,27 @@
+namespace System
+{
+    public partial class BinaryData
+    {
+        public BinaryData(byte[] data) { }
+        public BinaryData(object jsonSerializable, System.Text.Json.JsonSerializerOptions? options = null, System.Type? type = null) { }
+        public BinaryData(System.ReadOnlyMemory<byte> data) { }
+        public BinaryData(string data) { }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object? obj) { throw null; }
+        public static System.BinaryData FromBytes(byte[] data) { throw null; }
+        public static System.BinaryData FromBytes(System.ReadOnlyMemory<byte> data) { throw null; }
+        public static System.BinaryData FromObjectAsJson<T>(T jsonSerializable, System.Text.Json.JsonSerializerOptions? options = null) { throw null; }
+        public static System.BinaryData FromStream(System.IO.Stream stream) { throw null; }
+        public static System.Threading.Tasks.Task<System.BinaryData> FromStreamAsync(System.IO.Stream stream, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public static System.BinaryData FromString(string data) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static implicit operator System.ReadOnlyMemory<byte> (System.BinaryData data) { throw null; }
```

Comment:
Consider allowing these operators to support null _data_ args. That's how we've defined the implicit operators around [`ReadOnlyMemory<T>`](https://github.com/dotnet/runtime/blob/cb873bc51eb3e21e7cfba12bd3dac25518d85d67/src/libraries/System.Private.CoreLib/src/System/ReadOnlyMemory.cs#L118) and [`ReadOnlySpan<T>`](https://github.com/dotnet/runtime/blob/cb873bc51eb3e21e7cfba12bd3dac25518d85d67/src/libraries/System.Private.CoreLib/src/System/ReadOnlySpan.cs#L190).

Reply:
@KrzysztofCwalina - thoughts?

Reply:
If you do end up disallowing null _data_ args, remember to change the implementation to throw the proper exception. Currently it'll result in a null ref.

Reply:
The guideline is "DO NOT throw exceptions from implicit conversion operators."

So the implementations of these needs to check for null (since the null `byte[]` is the empty span, the null BinaryData can be one, too).